tools/mob_suite/mob_recon.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="mob_recon" name="MOB-Recon" version="1.4.1">
+  <description>(Type contigs and extract plasmid sequences)</description>
```

Comment:
This may be pedantic, but could you remove `( and )` from the description?

Reply:
Chaned and to plus

Reply:
Oh sorry, what I wrote was confusing.
You don't need the `(`, `)` characters at the beginning and end of your description element.

Reply:
Same as below.

Reply:
Ah, I see ). You meant to remove brackets from the description tag. OK)